content/changelog/2025-03-14.md

```diff
@@ -0,0 +1,68 @@
+---
+title: HIPAA compliance, GitHub Secret Scanning, and more
```

Comment:
maybe change the title to match the order of items in the changelog. Do we want to lead with HIPPA and then GitHub Secret Scanning (two security items together at the top?)